e2e/basic/main_test.go

```diff
@@ -2,19 +2,29 @@ package basic
 
 import (
 	"os"
-	"testing"
-
-	"github.com/sirupsen/logrus"
-
-	"github.com/celestiaorg/knuu/pkg/knuu"
 )
 
-func TestMain(m *testing.M) {
-	err := knuu.Initialize()
-	if err != nil {
-		logrus.Fatalf("error initializing knuu: %v", err)
+func convertViaMap(b bool) int {
+	table := map[bool]int{
+		true:  1,
+		false: 0,
 	}
-	logrus.Infof("Scope: %s", knuu.Scope())
-	exitVal := m.Run()
-	os.Exit(exitVal)
+	return table[b]
+}
```

Comment:
For something like this, it isn't worth allocating a map.
It is easier to just have a single if condition.
```go
func boolToExitCode(b bool) int {
        if b {
                return 1
        }
        return 0
}
```

Reply:
also I'm not sure why this is even needed? 
why are we using `os.Exit` in a test? 
Why aren't we just using `ts.Require().True(exitVal)` if that is what we are looking for?

Reply:
main will be deleted in another PR, we use test suite